<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistFilterWidget</class>
 <widget class="QWidget" name="PlaylistFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/icons.qrc">
    <normaloff>:/icons/24x24/file-m3u.png</normaloff>:/icons/24x24/file-m3u.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QPushButton" name="buttonSearch">
     <property name="text">
      <string>Search and filter</string>
     </property>
     <property name="icon">
      <iconset resource="../../../data/icons.qrc">
       <normaloff>:/icons/16x16/search.png</normaloff>:/icons/16x16/search.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonClear">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="../../../data/icons.qrc">
       <normaloff>:/icons/16x16/clear.png</normaloff>:/icons/16x16/clear.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="comboType">
     <item>
      <property name="text">
       <string>TV + Radio</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TV only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radio only</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QComboBox" name="comboLanguage">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QComboBox" name="comboCategory">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>editSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>editSearch</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonClear</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboLanguage</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>6</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboCategory</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
